index.js

```diff
@@ -0,0 +1,137 @@
+import { DateTime } from './modules/luxon';
+import Book from './modules/book';
+import {
+  container,
+  form,
+  bookTitle,
+  bookAuthor,
+  addBook,
+  bookContainer,
+  contactContainer,
+  list,
+  addNew,
+  contact,
+  pageTitle,
+} from './modules/access-html-elements';
+
+// creating book object from class Book
+class features {
+  static addNewBook = () => {
+    // js to create a new book object
+    const newBookTitle = bookTitle.value;
+    const newBookAuthor = bookAuthor.value;
+    const book = new Book(newBookTitle, newBookAuthor);
+
+    // to set up and update local storage
+    if (localStorage.getItem('awesome-books') == null) {
+      localStorage.setItem('awesome-books', '[]');
+    }
+
+    // getting old data and merging with new data
+    const oldData = JSON.parse(localStorage.getItem('awesome-books'));
+    oldData.push(book);
+
+    // updating local storage data
+    localStorage.setItem('awesome-books', JSON.stringify(oldData));
+  };
+
+  static removeBook = (index) => {
+    const books = JSON.parse(localStorage.getItem('awesome-books'));
+    books.splice(index, 1);
+    // js to update local storage after removing a book
+    localStorage.setItem('awesome-books', JSON.stringify(books));
+  };
+
+  static displayBooks = () => {
+    container.innerHTML = '';
+    const books = JSON.parse(localStorage.getItem('awesome-books'));
+    for (let i = 0; i < books.length; i += 1) {
+      const div = document.createElement('div');
+      div.className = i % 2 !== 0 ? 'white-color' : 'gray-color';
+
+      const textDiv = document.createElement('div');
+      textDiv.className = 'text-container';
+
+      const spanTitle = document.createElement('span');
+      spanTitle.className = 'book-title';
+      spanTitle.innerHTML = `"${books[i]?.title}"`;
+      textDiv.appendChild(spanTitle);
+
+      const by = document.createElement('span');
+      by.innerHTML = ' by ';
+      textDiv.appendChild(by);
+
+      const spanAuthor = document.createElement('span');
+      spanAuthor.className = 'author';
+      spanAuthor.innerHTML = books[i]?.author;
+      textDiv.appendChild(spanAuthor);
+
+      const removeButton = document.createElement('button');
+      removeButton.className = 'remove';
+      removeButton.innerHTML = 'Remove';
+      div.appendChild(removeButton);
+      removeButton.onclick = () => {
+        features.removeBook(i);
+        features.displayBooks();
+      };
+
+      div.appendChild(textDiv);
+      container.appendChild(div);
+    }
+  };
+}
```

Comment:
- [ ] [Optional] Well! It's great that you've added a feature class in the index.js file. However, considering that this project is focused on working with modules, it might be a good idea to move this class into a separate module file. Embracing the modular structure enhances the organization and maintainability of your codebase. Moreover, this approach creates room for more modules in the future, allowing for a more comprehensive and efficient codebase. I suggest you consider implementing this for better code practice. Keep up the good work! 🌟📝🚀